source/Grid/Grid.js

```diff
@@ -513,11 +517,11 @@ class Grid extends React.PureComponent<Props, State> {
     const totalColumnsWidth = this.state.instanceProps.columnSizeAndPositionManager.getTotalSize();
     const scrollLeft = Math.min(
       Math.max(0, totalColumnsWidth - width + scrollbarSize),
-      Math.max(0, this._scrollingContainer.scrollLeft + deltaX)
+      Math.max(0, this._scrollingContainer.scrollLeft + deltaX),
```

Comment:
? Confused why you added commans

Reply:
It's their prettier rules 🤷 bvaughn is apparently a sociopath who believes in trailing commas on lists